packages/core-components/src/components/TopBar.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+export interface TopBarItemType {
+  className: string;
+  linkContact1: string;
+  linkContact2: string;
+  phoneNumber1: string;
+  phoneNumber2: string;
+  nameContact1: string;
+  nameContact2: string;
+}
+
+export interface TopBarProps {
+  topBarItems: TopBarItemType[];
+  brandIcon: React.ReactNode;
+  phoneIcon: React.ReactNode;
+  languageButton: React.ReactNode;
+}
+
+const TopBar: React.FC<TopBarProps> = ({
+  topBarItems,
+  phoneIcon,
+  languageButton,
+}) => {
+  const itemsProps = topBarItems[0];
+  return (
+    <div
+      className={`${itemsProps.className} container sm:bg-white md:bg-primary flex flex-row items-center w-screen`}
+    >
+      <div className={'ContactIcon basis-8 shrink-0 ' + 'sm:ms-12 sm:ml-2 '}>
+        {phoneIcon}
+      </div>
+
+      <div
+        className={
+          'font-Roboto text-slate-200 border-e-2 font-medium ' +
+          'sm:invisible md:visible ' +
```

Comment:
do we have this component at small screen ?

Reply:
I heard that maybe we have to create new component top bar for mobile so I think we don't need this component at small screen size